### DISCUSSION TITLE
Проблемы с аимботом

### DISCUSSION THREAD

**Discussion by Pentagonov** [ttwizz/Open-Aimbot#104] [General]
Захожу в игру pubg simulator и когда врубаю аим то целится на игроков но при этом сама камера стоит на месте и не вращается, то есть срабатывает аим чисто визуально

**Comment by Pentagonov** [comment]
Как пофиксить?

**Reply by ttwizz** [reply]
Попробуйте использовать другой Aim Mode, это единое решение
